05_book_titles/book.rb

```diff
@@ -1,3 +1,24 @@
-class Book
 # write your code here
-end
+class Book
+    attr_accessor :title
```

Comment:
⛏️ just a nitpick! since you are defining your `title` setter method below, you only need to have the `attr_reader` macro. but if I were doing this assignment I'd probably just have used accessor too. :)